openid-4-verifiable-presentations-1_0.md

```diff
@@ -2852,6 +2867,91 @@ The following is a non-normative example of the unsecured payload of the Key Bin
 
 <{{examples/response/kb_jwt_unsecured.json}}
 
+### SD-JWT VCDM {#sd-jwt_vcdm}
```

Comment:
Consider renaming this to SD-JWT VCLD (or similar), to avoid further confusion with W3C VCDM.

Reply:
That makes a lot of sense indeed.

Reply:
SD-JWT VCLD ain't half bad

Reply:
> as SD-JWT VCDM already recognized by other SDO I strongly oppose against this change

In balance with a strong opposition, "other SDO" is a bit vague.

VCDM has become a non-ambiguous acronym that refers to a particular specification, namely W3C's Verifiable Credential Data Model. If anyone uses "VCDM" for something that doesn't comply with that spec, then they're making a mistake themselves, and certainly shouldn't be used as guidance by others for any naming choice.

Reply:
Even better would be "SD-JWT DCLD", since "VC" also refers to a particular W3C specification :)